src/modules/navigator/gpsfailure_params.c

```diff
@@ -56,12 +51,12 @@
  * @increment 1
  * @group GPS Failure Navigation
  */
-PARAM_DEFINE_FLOAT(NAV_GPSF_LT, 30.0f);
+PARAM_DEFINE_FLOAT(NAV_GPSF_LT, 0.0f);
```

Comment:
This is a drastic change. Wouldn't it be a sane default to have some loiter time before going into termination?

Reply:
Actually my intent is the opposite, by default continue loitering with no throttle all the way to the ground. If someone wants terminate they'd have to enable it.

Reply:
Oh, got it!